gateway/api/access_policies/users.py

```diff
@@ -0,0 +1,35 @@
+"""
+Access policies implementation for Users access
+"""
+
+import logging
+from django.contrib.auth.models import AbstractUser
+
+
+logger = logging.getLogger("gateway")
+
+
+class UserAccessPolicies:
+    """
+    The main objective of this class is to manage the access for the user
+    to the application.
+    """
+
+    @staticmethod
+    def can_access(user: type[AbstractUser]) -> bool:
+        """
+        Checks if the user has access to the application.
+        For that it will need to be active.
```

Comment:
Small clarification (I wasn't sure if it was referring to the user or the application until I saw the code):

```suggestion
        For that, the user will need to be active.
```